src/pages/demo.tsx

```diff
@@ -4,10 +4,7 @@ import Layout from '@theme/Layout';
 export default function Demo() {
   return (
     <Layout title="Forminer demo" wrapperClassName="demo-container">
-      <iframe
-        title="Forminer demo"
-        src="https://vazco.github.io/form-builder-project/"
-      />
+      <iframe title="Forminer demo" src="https://forminer.com" />
```

Comment:
The demo is most likely hosted on some other domain, right?

Reply:
My bad, you're totally right! Honestly, I'm not sure this topic has ever been discussed  :thinking: